src/client/components/DetailsModal/index.tsx

```diff
@@ -228,109 +227,108 @@ export const DetailsModal = ({
 
-  const isUnderwritingGuidelinesHit = isLimitHit(createQuoteBundleData)
-  const isQuoteCreationFailed = isQuoteBundleError(createQuoteBundleData)
+  const isUnderwritingGuidelinesHit = getLimitsHitFromEditQuoteMutation(
+    editQuoteData,
+  ).length
+  const isQuoteCreationFailed = isQuoteBundleError(editQuoteData)
 
-  const reCreateQuoteBundle = (form: QuoteInput) => {
+  const editQuotes = async (form: QuoteInput) => {
     // cleanup null form fields
     Object.keys(form).forEach(
       (k) =>
         form[k as keyof QuoteInput] == null &&
         delete form[k as keyof QuoteInput],
     )
 
-    const newQuotes = allQuotes.map((quote) => {
-      const { startDate, currentInsurer, data } = quote
-      const quoteData = Object.keys(data).reduce<
-        Partial<QuoteDetailsInput & QuoteDataCommon>
-      >((acc, key) => {
-        switch (key) {
-          case 'numberCoInsured':
-            acc[key] =
-              typeof form.data.householdSize === 'number'
-                ? form.data.householdSize - 1
-                : data[key]
-            return acc
-          case 'livingSpace':
-          case 'squareMeters':
-            acc.squareMeters =
-              form.data['livingSpace'] ||
-              form.data['squareMeters'] ||
-              data['livingSpace'] ||
-              data['squareMeters']
-            acc.livingSpace =
-              form.data['livingSpace'] ||
-              form.data['squareMeters'] ||
-              data['livingSpace'] ||
-              data['squareMeters']
-            return acc
-          case 'subType':
-            if (marketLabel === 'SE') {
-              acc[key] =
-                mainQuote.id === quote.id
-                  ? getSubType(form.data)
-                  : getSubType(data)
-            } else {
+    for (const quote of allQuotes) {
+      const { id, data } = quote
+      const payload = {
+        ...form,
+        data: Object.keys(data).reduce<
+          Partial<QuoteDetailsInput & QuoteDataCommon>
+        >((acc, key) => {
+          switch (key) {
+            case 'numberCoInsured':
               acc[key] =
-                mainQuote.id === quote.id
-                  ? form.data['subType']
-                  : data['subType']
+                typeof form.data.householdSize === 'number'
+                  ? form.data.householdSize - 1
+                  : data[key]
+              return acc
+            case 'livingSpace':
+            case 'squareMeters':
+              acc.squareMeters =
+                form.data['livingSpace'] ||
+                form.data['squareMeters'] ||
+                data['livingSpace'] ||
+                data['squareMeters']
+              acc.livingSpace =
+                form.data['livingSpace'] ||
+                form.data['squareMeters'] ||
+                data['livingSpace'] ||
+                data['squareMeters']
+              return acc
+            case 'subType':
+              if (marketLabel === 'SE') {
```

Comment:
Can you comment what's difference between SE and other markets?

Reply:
It seems we configure `subType` differently for SE. No idea why tho. Do you know @robinandeer ?